pkg/plugins/manager/loader/finder/local_test.go

```diff
@@ -176,40 +177,19 @@ func TestFinder_Find(t *testing.T) {
 		{
 			name:       "Multiple plugin dirs",
 			pluginDirs: []string{"../../testdata/duplicate-plugins", "../../testdata/invalid-v1-signature"},
-			expectedBundles: []*plugins.FoundBundle{{
-				Primary: plugins.FoundPlugin{
-					JSONData: plugins.JSONData{
-						ID:   "test-app",
-						Type: plugins.TypeDataSource,
-						Name: "Parent",
-						Info: plugins.Info{
-							Author: plugins.InfoLink{
-								Name: "Grafana Labs",
-								URL:  "http://grafana.com",
-							},
-							Description: "Parent plugin",
-							Version:     "1.0.0",
-							Updated:     "2020-10-20",
-						},
-						Dependencies: plugins.Dependencies{
-							GrafanaVersion: "*",
-							Plugins:        []plugins.Dependency{},
-						},
-					},
-					FS: mustNewStaticFSForTests(t, filepath.Join(testData, "duplicate-plugins/nested")),
-				},
-				Children: []*plugins.FoundPlugin{
-					{
+			expectedBundles: []*plugins.FoundBundle{
```

Comment:
are the changes in this file expected?

It seems odd the test was previously expecting a parent and now expecting a child. What happened?